src/pages/auth/signup/signup.jsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
```

Comment:
UI suggestion: keeping the checkbox and text in the same line looks good.
![Screenshot 2022-03-29 at 6 18 53 PM](https://user-images.githubusercontent.com/79623546/160614817-092fab77-9e48-4d01-9102-f94283649a17.png)